### DISCUSSION TITLE
PatchTool 6.1 4/15 (Fix for 4/13)

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#135] [Announcements]
修正v2.58有機率造成dx11錯誤的問題。

- FFXIVChnTextPatch-0415.rar：本體。一般而言，下載這個即可。Backup資料夾內含乾淨檔案，可視情況取用。

選用：
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- rawexd-zhcn-0415.rar：簡體中文CSV。解壓後改名為rawexd放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- backup-0415.rar：乾淨的0x0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原，然後再次嘗試漢化。
- rawexd-ja-610-v2-0415.rar：日文原文的CSV。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.59'>PatchTool 6.1 4/15 (Fix for 4/13)</a>.</em>

**Comment by Bally0886** [comment]
謝謝大大的努力!

**Comment by JerryTsai73** [comment]
感謝大大

**Comment by maaapp56** [comment]
辛苦啦

**Comment by williamhong1021** [comment]
謝謝大大的努力與辛苦

**Comment by siler2011** [comment]
感謝大大!!

**Comment by cpureaz3958** [comment]
謝謝大大 !!!

**Comment by earvinwu** [comment]
辛苦了！

**Comment by NightOneOne911** [comment]
寫謝大大的辛苦

**Comment by IamkennyTW** [comment]
感謝大大

**Comment by cydad111** [comment]
感謝了.

**Comment by jiyuanfu** [comment]
感谢大佬，点星感谢

**Comment by qwe3410475** [comment]
辛苦了~

**Comment by bluesjamgt** [comment]
thx with love :>

**Comment by nonyumeya** [comment]
辛苦了 感謝 :>

**Comment by jiyuanfu** [comment]
您好，谢谢您的来信，我已收到。傅纪元。
